core/src/main/java/org/elasticsearch/gateway/AsyncShardFetch.java

```diff
@@ -57,24 +59,24 @@
     /**
      * An action that lists the relevant shard data that needs to be fetched.
      */
-    public interface List<NodesResponse extends BaseNodesResponse<NodeResponse>, NodeResponse extends BaseNodeResponse> {
+    public interface Lister<NodesResponse extends BaseNodesResponse<NodeResponse>, NodeResponse extends BaseNodeResponse> {
```

Comment:
Just to be clear for me, you were referring to the need to rename this, right? :)